app/src/main/java/org/liberty/android/fantastischmemo/ui/RecentListFragment.java

```diff
@@ -88,6 +107,26 @@ public void onAttach(Context activity) {
         super.onAttach(activity);
         mActivity = (Activity) activity;
         setHasOptionsMenu(true);
+        LocalBroadcastManager.getInstance(mActivity).registerReceiver(mClearSelectionReceiver,
+                new IntentFilter("recent-list-clear-selection"));
```

Comment:
Could you make the intent action as a constant that is access both here and in the adapter?

You can put it in the Util you created.

public static final String RECENT_LIST_CLEAR_SELECTION = "recent-list-clear-selection";